airflow/example_dags/tutorial_taskflow_api_etl_virtualenv.py

```diff
@@ -19,24 +19,14 @@
 
 # [START tutorial]
 # [START import_module]
-import json
```

Comment:
Why move this?

Reply:
Unfortunately the "extract" task fails without it.
```[2021-08-06, 13:26:50 UTC] {process_utils.py:135} INFO - Executing cmd: /tmp/venvys6_yc45/bin/python /tmp/venvys6_yc45/script.py /tmp/venvys6_yc45/script.in /tmp/venvys6_yc45/script.out /tmp/venvys6_yc45/string_args.txt
[2021-08-06, 13:26:50 UTC] {process_utils.py:139} INFO - Output:
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO - Traceback (most recent call last):
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO -   File "/tmp/venvys6_yc45/script.py", line 40, in <module>
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO -     res = extract(*arg_dict["args"], **arg_dict["kwargs"])
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO -   File "/tmp/venvys6_yc45/script.py", line 37, in extract
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO -     order_data_dict = json.loads(data_string)
[2021-08-06, 13:26:50 UTC] {process_utils.py:143} INFO - NameError: name 'json' is not defined
```